src/SqlAsyncCollector.cs

```diff
@@ -292,6 +301,18 @@ private async Task UpsertRowsAsync(IList<T> rows, SqlAttribute attribute, IConfi
             }
         }
 
+        private static bool IsUnsupportedType(string property, IDictionary<string, string> columns)
+        {
+            foreach (string unsupportedType in UnsupportedTypes)
+            {
+                if (columns[property].StartsWith(unsupportedType, StringComparison.OrdinalIgnoreCase))
```

Comment:
Is there a reason to use StartsWith instead of comparing the whole string?

Reply:
The column type also includes the size, for example text(10).

Reply:
Can you have sizes for these types? I tried and it gave me an error

![image](https://user-images.githubusercontent.com/28519865/217589639-7817d97a-57b1-454a-8578-4106f23e165e.png)

Reply:
Hm, the value I got while testing was text(*) so I thought there can be sizes. Maybe the column type returned will always be text(*), ntext(*), and image(*). I'll double check.

Reply:
The `(*)` gets added on because of this line in the query https://github.com/Azure/azure-functions-sql-extension/blob/61ca7215d75e8c6e092aabcea3f75a17f7d7f059/src/SqlAsyncCollector.cs#L511 I updated the UnsupportedTypes to include the `(*)` and simplified the check.